gimme_aws_creds/config.py

```diff
@@ -211,6 +211,7 @@ def update_config_file(self):
                 okta_username = Okta username
                 aws_default_duration = Default AWS session duration (3600)
                 preferred_mfa_type = Select this MFA device type automatically
+                include_path - (optional) includes that full role path to the role name for profile
```

Comment:
Change the formatting and wording here just a little (hopefully this is clearer):
```
include_path - (optional) Includes full role path to the role name in AWS credential profile name
```

Reply:
Will do